src/components/Payroll/PayrollEditEmployee/PayrollEditEmployeePresentation.tsx

```diff
@@ -9,21 +11,23 @@ interface PayrollEditEmployeeProps {
 
 export const PayrollEditEmployeePresentation = ({ onDone }: PayrollEditEmployeeProps) => {
   const { Button, Heading, Text } = useComponentContext()
+  useI18n('Payroll.PayrollEditEmployee')
+  const { t } = useTranslation('Payroll.PayrollEditEmployee')
   const formHandlers = useForm()
   return (
     <Flex flexDirection="column" gap={20}>
-      <Heading as="h2">{`Edit Hannah Arendt's payroll`}</Heading>
+      <Heading as="h2">{t('pageTitle', { employeeName: 'Hannah Arendt' })}</Heading>
       <Heading as="h1">$1,173.08</Heading>
-      <Text>Gross pay</Text>
-      <Heading as="h3">Regular hours</Heading>
+      <Text>{t('labels.grossPay')}</Text>
+      <Heading as="h3">{t('labels.regularHours')}</Heading>
       <FormProvider {...formHandlers}>
         <Form>
-          <NumberInputField defaultValue={40} isRequired label="Hours" name="hours" />
+          <NumberInputField defaultValue={40} isRequired label={t('labels.hours')} name="hours" />
         </Form>
       </FormProvider>
 
-      <Button onClick={onDone} title="Done">
-        Done
+      <Button onClick={onDone} title={t('buttons.doneTitle')}>
+        {t('buttons.done')}
```

Comment:
Can we just remove the translations added to this file? i have it implemented for the Edit ticket in progress and it'll just result in merge conflicts 😅